kuksa_viss_client/__main__.py

```diff
@@ -164,6 +183,36 @@ def do_getValue(self, args):
             print(highlight(resp, lexers.JsonLexer(), formatters.TerminalFormatter()))
         self.pathCompletionItems = []
 
+    @with_category(VISS_COMMANDS)
+    @with_argparser(ap_subscribe)
+    def do_subscribe(self, args):
+        """Subscribe the value of a path"""
+        if self.checkConnection():
+            resp = self.commThread.subscribe(args.Path, lambda msg: self.subscribeCallback(args.Path, msg))
+            resJson =  json.loads(resp) 
+            if "subscriptionId" in resJson:
+                fileName = os.getcwd() + "/log_"+args.Path+"_"+str(time.time())
+                self.subscribeFileDesc[args.Path] = open(fileName, "w")
+                self.subscribeIdToPath[resJson["subscriptionId"]] = args.Path
+                print("Subscription log available at " + fileName)
+                print("Execute tail -f " + fileName + " on another Terminal instance")
```

Comment:
Is it possible to ASK first, whether to open an xterm? Because often there might not be an X system on the system, and this would fail

Reply:
Ofcourse we can add a flag for the user to use (or not use) xterm.

Currently it would fail rather silently. 
![image](https://user-images.githubusercontent.com/69036497/165310449-4ced6278-a192-474f-9997-4e76af895f83.png)
If you like, we can even redirect the stderr of the subprocess to avoid this.
This way user would not see this at all. The client would use xterm if it is available. Else the user needs to execute the corresponding tail command.

What would be a better solution?